VirtualAssistantBusinessLogic/VirtualAssistantBusinessLogicTests/SparQL/SparQLWhereTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using NSubstitute;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using VirtualAssistantBusinessLogic.KnowledgeGraph;
+using VirtualAssistantBusinessLogic.SparQL;
+
+namespace VirtualAssistantBusinessLogicTests.SparQL
+{
+    [TestFixture]
+    public class SparQLWhereTests
+    {
+        private SparQLSelect subSparQLSelect;
+        private ISPOEncoder subSPOEncoder;
+
+        [SetUp]
+        public void SetUp()
+        {
+            EncodedSPO output = new($"?s1 ?p \"SUBJECT\"@en . ", $"?s1");
```

Comment:
Er der grund til $ foran strings her?
```suggestion
            EncodedSPO output = new($"?s1 ?p \"SUBJECT\"@en . ", "?s1");
```